pkgs/applications/networking/shellhub-agent/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "shellhub-agent";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "shellhub-io";
+    repo = "shellhub";
+    sha256 = "0cd41ing1pcf1bdaaq00w5h7lih5j2kcaa0m41g3ikm3vd1w5qna";
+  };
+
+  patches = [ ./go-mod-tidy.patch ];
```

Comment:
It would be great if you could upstream those changes and then reference the patch from the commit hash in the PR. Like https://github.com/NixOS/nixpkgs/commit/51b455552402934e7a3abd49e843e46774078664.patch.

Reply:
We are working on the 0.5.0 release and I'll update it once it is released, dropping this patch.